truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -82,6 +83,28 @@ def copy_into_build_dir(from_path: Path, path_in_build_dir: str):
         # Download external data
         download_external_data(self._spec.external_data, data_dir)
 
+        # Download from HuggingFace
+        model_files = {}
+        if config.hf_cache:
+            (build_dir / "cache_warmer.py").write_text(
+                """
+if __name__ == "__main__":
+    from huggingface_hub import hf_hub_download
+    import sys
+    file_name = sys.argv[1]
+    repo_name = sys.argv[2]
+    revision_name = sys.argv[3]
+    hf_hub_download(repo_name, file_name)
```

Comment:
you need to pass the revision_name to the download here

Reply:
we should have the `revision` optional to match the `hf_hub_download` interface. nit: let's just use the param names (`repo_id`, `filename`) they have [here](https://huggingface.co/docs/huggingface_hub/v0.16.3/en/package_reference/file_download#huggingface_hub.hf_hub_download) for clarity

Reply:
^ looked at config level changes – the defaults there should take care of the optionality